language-server/package.json

```
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "some-sass-language-server": "^2.0.2"
+  }
```

Set `name` and `private` to avoid some warnings:

```suggestion
  "name": "some-sass",
  "private": true,
  "dependencies": {
    "some-sass-language-server": "^2.0.2"
  }
```

---

Also run `npm i` to update the lock file.

---

Done